### PR TITLE
feat:added footer to video library

### PR DESCRIPTION
**Pull request by mariyasada**
added footer with social media icons to video library
footernew

### CODE REVIEW

File: src/Components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { BsGithub,BsTwitter,BsLinkedin } from "react-icons/bs";
+import "../Footer/Footer.css";
```
Comment (tusharkandpal) [inline src/Components/Footer/Footer.jsx]: You can directly import css using `./Footer.css`

File: src/Pages/Home/Home.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { HomePageCard, Categories } from "../../Components";
+import { HomePageCard, Categories, Footer } from "../../Components";
 import "../Home/Home.css";
```
Comment (tusharkandpal) [inline src/Pages/Home/Home.jsx]: Same here! Please correct

**Review by tusharkandpal — APPROVED** [review]
Can keep `Footer` component out of routes if common for all screens.

Clean & export from single file is a good choice.

**Review by JantuDeb — APPROVED** [review]
Footer component looks good, I would suggest you to add some internal links such as categories,

**Review by Bhakti199 — APPROVED** [review]
This code looks fine to me.
Like the idea of creating small components.
This PR is good to merge

**Pull request merged**